src/utils.rs

```diff
@@ -75,6 +75,10 @@ pub fn gen_hex_string(rng: &mut ThreadRng, length: u64) -> String {
     gen_string_with_chars(rng, "0123456789abcdef", length)
 }
 
+pub fn gen_sha256_string(rng: &mut ThreadRng) -> String {
+    gen_hex_string(rng, 64)
```

Comment:
I'm not sure specializing this call is really worth this function. Why not just always call `gen_hex_string(rng, 64)` in place? It only has a documentary effect here as it doesn't really save any repeated code. In this case, I'd actually recommend you specialize it using a lambda inside `docker.rs` like this: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=bcccccfa7650952efa0ee65422c3cc68

I hope that gets the point across.